src/Model/Endpoint.php

```diff
@@ -153,6 +160,9 @@ public function __construct(array $config = [])
         if (!empty($config['resourceClass'])) {
             $this->resourceClass($config['resourceClass']);
         }
+        if (!empty($config['inflection'])) {
```

Comment:
Keep the option name `inflect` so that it's consistent with config name used in route builder.

Reply:
thanks for the clarification!